File: api/DTOs/Admin/Orders/OrderItemToFwdDto.cs
using api.Entities.Admin.Order;

namespace api.DTOs.Admin.Orders
{
    public class OrderItemToFwdDto
    {
        public int Id {get; set;}
        public int OrderItemId { get; set; }
        public int OrderId { get; set; }
        public DateTime OrderDate { get; set; }
        
        public int CategoryId { get; set; }
        public string CategoryRef { get; set; }
        public string CategoryName {get; set;}
        public int Quantity {get; set;}
        public ICollection<OrderForwardCategoryOfficial> DLForwardDates { get; set; }
    }
}

File: client/src/app/_dtos/admin/orderItemToFwdDto.ts
import { IOrderForwardDate } from "src/app/_models/orders/orderForwardDate";

export interface IOrderItemToFwdDto
{
     id: number;
     orderItemId: number;
     orderId: number;
     orderDate: Date;

     categoryId: number;
     categoryRef: string;
     categoryName: string;
     quantity: number;
     charges: number;
     dlForwardDates: IOrderForwardDate[];
}

export class OrderItemToFwdDto implements IOrderItemToFwdDto
{
     id= 0;
     orderItemId= 0;
     orderId= 0;
     orderDate= new Date('1900-01-01');
     
     categoryId= 0;
     categoryRef= '';
     categoryName= '';
     quantity= 0;
     charges= 0;
     dlForwardDates: IOrderForwardDate[]=[];
}